benchmark/elastic/elastic_benchmark_config.py

```diff
@@ -0,0 +1,19 @@
+DATA_DIR="workload/trace/test/"
```

Comment:
Shouldn't this directory be ../workload/trace/? Do you assume the elastic script is run from the benchmark directory or the elastic directory?